env_spec.py

```diff
@@ -0,0 +1,87 @@
+import re
+
+def check_latin_chars(input_str):
+    """
+    function that checks if input_str consists of latin chars 
+    (input_str can contain underscore and numeric digits)
+    """
+    try:
+        input_str.encode('utf-8').decode('ascii')
+    except Exception:
+        return False
+    else:
+        return True
+
+
+def check_symbols(input_str):
+    """
+    checks input_str (must be digit/underscore/uppercase latin char)
+    return False for error about specifications or True if correct
+    """
+    alphanumeric_that_does_not_start_with_digit = r'^[A-Z_][0-9A-Z_]*$'
+
+    ret = re.match(alphanumeric_that_does_not_start_with_digit, input_str)
+
+    if ret == None:
+        return False
+    else:
+        return True
+
+
+def check_input(field_array):
+    """
+    Checks syntax of input(uppercase latin chrars,numeric digits, underscores).
+    Returns False for error or True for correct syntax.
+    """
+    for line in field_array:       
+        
+        latin_chars = check_latin_chars(line)
```

Comment:
We do not have to check for latin characters. [The regular expression](https://github.com/sourcelair/env-spec-py/pull/2/files#diff-662ae986ce9e28e035956190e3f42976R21) will do that for us 😄!